packages/x-charts/src/internals/plugins/featurePlugins/useChartInteraction/useChartInteraction.types.ts

```diff
@@ -30,7 +34,7 @@ export interface UseChartInteractionState {
     /**
      * The item currently interacting.
      */
-    item: null | ChartItemIdentifier<ChartSeriesType>;
+    item: null | ChartItemIdentifierWithData<ChartSeriesType>;
```

Comment:
I should probably be like that when we add control. But for now it's fine as it

```suggestion
    item: null | ChartItemIdentifier<ChartSeriesType> | ChartItemIdentifierWithData<ChartSeriesType>;
```